### DISCUSSION TITLE
Where is Windows ZIP?

### DISCUSSION THREAD

**Discussion by wobbo** [royshil/obs-backgroundremoval#409] [General]
I can only find the EXE. 
Where is the ZIP?

obs-backgroundremoval-1.1.2-windows-x64**.zip**

**Comment by umireon** [comment]
Zip will be released from the next release.

**Reply by wobbo** [reply]
Thanks!

**Comment by royshil** [comment]
Indeed
We will release v1.1.3 shortly
